api.go

```diff
@@ -698,3 +698,44 @@ func KillContainer(podID, containerID string, signal syscall.Signal, all bool) e
 
 	return nil
 }
+
+// PsContainer is the virtcontainers entry point to list
+// processes running inside a container
+func PsContainer(podID, containerID string, psArgs []string) error {
```

Comment:
Maybe I am missing something here but you never return anything, right ? How do you plan to get the data back ?

Also I think it could be part of `StatusContainer()` instead of adding a new function to the exported API. @sameo any thoughts on this one ?